.github/workflows/end-to-end-tests.yaml

```diff
@@ -133,6 +133,11 @@ jobs:
             deployments: write
         steps:
             - uses: actions/checkout@v4
+              with:
+                  ref: ${{ steps.sha.outputs.sha }}
+                  persist-credentials: false
+                  path: matrix-react-sdk
```

Comment:
checking out in a subdirectory would mean all later steps need to cd into it

Reply:
Yeah, presumably just using the default dir is fine?

Reply:
Yup!